data/json/items/comestibles/brewing.json

```diff
@@ -0,0 +1,345 @@
+[
+    {
+        "type" : "COMESTIBLE",
+        "id" : "brew_whiskey",
+        "name" : "whiskey wort",
+        "description" : "Unfermented whiskey.  The base of a fine drink.  Not the traditional preparation, but you don't have the time.",
+        "weight" : 33,
+        "color" : "brown",
+        "container" : "bottle_glass",
+        "symbol" : "~",
+        "nutrition" : 2,
+        "quench" : 4,
+        "fun" : -15,
+        "price" : 0,
+        "volume" : 1,
+        "charges" : 7,
+        "phase" : "liquid",
+        "comestible_type" : "DRINK",
+        "brewable" : {
+            "time" : 7200,
+            "results" : [ "wash_whiskey", "yeast" ]
+        }
+    },
+    {
+        "type" : "COMESTIBLE",
+        "id" : "wash_whiskey",
+        "name" : "whiskey wash",
+        "name_plural" : "whiskey washes",
+        "description" : "Fermented, but not distilled whiskey.  No longer tastes sweet.",
+        "weight" : 33,
+        "color" : "brown",
+        "addiction_type" : "alcohol",
+        "addiction_potential" : 1,
+        "use_action" : "ALCOHOL_WEAK",
+        "stim" : -2,
+        "container" : "jug_plastic",
+        "symbol" : "~",
+        "nutrition" : 1,
+        "quench" : 2,
+        "healthy" : -2,
+        "fun" : 2,
+        "material" : [ "water", "alcohol" ],
+        "volume" : 1,
+        "charges" : 7,
+        "phase" : "liquid",
+        "comestible_type" : "DRINK"
+    },
+    {
+        "type" : "COMESTIBLE",
+        "id" : "brew_vodka",
+        "name" : "vodka wort",
+        "description" : "Unfermented vodka.  Water with sugar from enzymatic breakdown of malted grains or just added in pure form.",
+        "weight" : 50,
+        "color" : "light_cyan",
+        "container" : "bottle_glass",
+        "symbol" : "~",
+        "nutrition" : 2,
+        "quench" : 4,
+        "fun" : -5,
+        "price" : 0,
+        "volume" : 1,
+        "charges" : 7,
+        "phase" : "liquid",
+        "comestible_type" : "DRINK",
+        "brewable" : {
+            "time" : 3600,
+            "results" : [ "wash_vodka", "yeast" ]
+        }
+    },
+    {
+        "type" : "COMESTIBLE",
+        "id" : "wash_vodka",
```

Comment:
Could `copy-from` an `abstract` type, for example `alcohol_wash` which would cut out a lot of the repetition